app-min/test__if_not_no-unit-tests/my-app.spec.ext

```diff
@@ -1,38 +1,19 @@
-// @if mocha
-import { expect } from 'chai';
-// @endif
 // @if vitest
-import { describe, it, expect } from 'vitest';
-// @endif
-// @if shadow-dom
-import { CustomElement } from 'aurelia';
+import { describe, it } from 'vitest';
 // @endif
-import { render } from './helper';
 import { MyApp } from '../src/my-app';
+import { createFixture } from '@aurelia/testing';
+import './setup';
```

Comment:
vitest/bun/jest allows a `setupFiles` config (`preload` for bun though) to run these kind of scripts. Do we want to use that or do we want to keep it consistent across all of our setups?

Reply:
I was worried about setupFiles does not play well with before/after hooks. Will try.

Reply:
Jest doesn't play well with beforeAll in setupFiles :(

```
> jest

FAIL test/my-app.spec.ts
  ● Test suite failed to run

    ReferenceError: beforeAll is not defined

      10 |
      11 | const fixtures: IFixture<object>[] = [];
    > 12 | beforeAll(() => {
         | ^
      13 |   bootstrapTextEnv();
      14 |   onFixtureCreated(fixture => {
      15 |     fixtures.push(fixture);

      at Object.<anonymous> (test/setup.ts:12:1)
```

Reply:
vitest works well.

Reply:
Got it, jest has another setupFilesAfterEnv. Crazy.